scripts/ci/run_ci.sh

```diff
@@ -269,6 +275,10 @@ if [ -n "$main_ci" ]; then
 	cat test_file_3.txt test_file_2_in.txt test_file_1_in.txt > test_file.txt
 
 	# Run a subset of tests based on matrix size
+    if [ "$matrix" = "1" ]; then
```

Comment:
same here, please

Reply:
not sure, why it is displayed like this here. For both the places indentation looks fine on my local as well as in the forked repository.

Reply:
you are mixing tabs and spaces, you should use tabs here.

Reply:
Fixed, Thanks for noticing this.